proxy/files/bin/proxy-start.sh

```diff
@@ -21,7 +22,18 @@ if [ "${INTERNAL_GW_URL:-X}" = 'X' ]; then
   exit 1
 fi
 
-erb /etc/nginx/conf.d/evaka-nginx.conf.template > /etc/nginx/conf.d/evaka-nginx.conf
+if test -n "$DEPLOYMENT_BUCKET"; then
+  s3download "$DEPLOYMENT_BUCKET" "nginx-proxy" /etc/nginx/
```

Comment:
Nit: Naming the S3 "directory" just "proxy" would be more consistent with the other names